odl/contrib/solvers/spdhg/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Contributed code for the stochastic PDHG."""
```

Comment:
Please add a reference to the article here, it would be nice for users to see where it comes from! (Also free PR for you :smile:)

Reply:
done